src/main/scala/com/monsanto/arch/cloudformation/model/resource/AutoScaling.scala

```diff
@@ -44,9 +45,45 @@ case class `AWS::AutoScaling::LaunchConfiguration`(
 }
 
 object `AWS::AutoScaling::LaunchConfiguration` extends DefaultJsonProtocol {
-  implicit val format: JsonFormat[`AWS::AutoScaling::LaunchConfiguration`] = jsonFormat9(`AWS::AutoScaling::LaunchConfiguration`.apply)
+  implicit val format: JsonFormat[`AWS::AutoScaling::LaunchConfiguration`] = jsonFormat10(`AWS::AutoScaling::LaunchConfiguration`.apply)
+}
+case class BlockDeviceMapping(
+  DeviceName:   Token[String],
+  Ebs:  Option[AutoScalingEBS] = None,
+  NoDevice: Option[Token[Boolean]] = None,
+  VirtualName: Option[Token[String]] = None
+)
+object BlockDeviceMapping extends DefaultJsonProtocol {
+  implicit val format: JsonFormat[BlockDeviceMapping] = jsonFormat4(BlockDeviceMapping.apply)
+}
+
+case class AutoScalingEBS(
+   DeleteOnTermination: Option[Token[Boolean]] = None,
+   Iops: Option[Token[Int]] = None,
```

Comment:
Token[Int] works fine -- I've used the one on VolumeSize already.  It's used numerous places in CFTG.  Token[Boolean] should work too -- I haven't tested it though since I don't know why you'd even want to use that parameter, really.  It was already being used in other Autoscaling parameters.